src/legacy/ui/public/visualize/loader/pipeline_helpers/utilities.ts

```diff
@@ -48,10 +47,12 @@ function isTermsFieldFormat(
 const config = chrome.getUiSettingsClient();
 
 const getConfig = (...args: any[]): any => config.get(...args);
-const getDefaultFieldFormat = () => ({ convert: identity });
+const getDefaultFieldFormat = () => (({ convert: identity } as unknown) as FieldFormat);
```

Comment:
Can we use FieldFormat.from(indentity) here?

Reply:
Changed